DESCRIPTION

```diff
@@ -8,25 +8,25 @@ Description: The Spectra package defines an efficient infrastructure
    backends tuned for fast data access and processing and backends for very
    large data sets ensuring a small memory footprint.
 Authors@R: c(person(given = "RforMassSpectrometry Package Maintainer",
-		    email = "maintainer@rformassspectrometry.org",
```

Comment:
you need also to bump the version above to 1.1.19

Reply:
Ah yes, forgot that! Done now, thanks